frontend/src/scenes/ingestion/v2/types.ts

```diff
@@ -6,9 +6,15 @@ import {
     WEB,
     webFrameworks,
     THIRD_PARTY,
-    BOOKMARKLET,
+    GENERATING_DEMO_DATA,
 } from 'scenes/ingestion/v2/constants'
 
 export type Framework = keyof typeof webFrameworks | keyof typeof mobileFrameworks | typeof API | null
 
-export type PlatformType = typeof WEB | typeof MOBILE | typeof BACKEND | typeof THIRD_PARTY | typeof BOOKMARKLET | null
+export type PlatformType =
+    | typeof WEB
+    | typeof MOBILE
+    | typeof BACKEND
+    | typeof THIRD_PARTY
+    | typeof GENERATING_DEMO_DATA
```

Comment:
It's not a platform though, why does it need to be here?

Reply:
Ha, because that's where the bookmarklet was. I'll move it.

Reply:
> > Why three main CTA? Can't we have the second and third buttons as secondary?
> 
> @kappa90 I didn't want to make people feel like one option was more "correct" than another. We could use their "what is your role?" answer to highlight the one that makes most sense for them..

Yes to using the role, but by default I'd say the first option is highlighted.